taas/config.yml

```diff
@@ -0,0 +1,44 @@
+---
+# TODO: We should have a config.d or similar setup; this means adding
+#       additional services doesn't result in them fighting for control of the
+#       same config file.
+
+sources:
+    # This maps directly to the API URL
+    functional_roles:
+        v1:
+            # Extracted from the spreadsheet URL
+            key: 1c9wehuauQAAegElIRI6vhWktKSI-PcPjHHiXdqASonk
+
+            # Sheet number. Extract from the spreadsheet URL as well.
+            gid: 0
+
+            # Mapping of column names -> API fields
+            # Columns that aren't mentioned get ignored.
+            mapping:
+                "@context":
+                    type: literal
+                    value: https://raw.githubusercontent.com/UN-OCHA/taas-data/master/json/v1/functional_roles.jsonld
+                "@id":
+                    type: concat
+                    prefix: http://vocabulary.unocha.org/FunctionalRole/
+                    field: ID
+
+                    # XXX: This shouldn't be optional, but we're missing an
```

Comment:
Are we trying to distance ourself from the datasets? I'm a sucker for inclusive language to try and foster shared responsibility, camaraderie, and especially new contributors who will come and fix my code. :)

Done.